SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveBaseCampaign.java

```diff
@@ -82,6 +77,9 @@ public SwrveBaseCampaign(SwrveBase<?, ?> controller, JSONObject campaignData) th
         setTalkController(controller);
         SwrveLogger.i(LOG_TAG, "Loading campaign " + getId());
 
+        // Start with an empty state
+        this.state = new SwrveCampaignState();
```

Comment:
It is inside the campaign and we know it is ours, so I don't think we need to add swrve or campaign to it